mldsa/fips202/fips202x4.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) The mlkem-native project authors
+ * Copyright (c) The mldsa-native project authors
+ * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+ */
+
+/* References
+ * ==========
+ *
+ * - [FIPS203]
+ *   FIPS 203 Module-Lattice-Based Key-Encapsulation Mechanism Standard
+ *   National Institute of Standards and Technology
+ *   https://csrc.nist.gov/pubs/fips/203/final
+ */
```

Comment:
Should we add FIPS 204 here?

Reply:
These references are auto-generated in mlkem-native. I've removed it for now.

We don't have stack zeroization yet. Once we add it we can add an appropriate reference and add the autogeneration of references.